sql-jdbc/src/main/java/com/amazon/opendistroforelasticsearch/jdbc/types/ElasticsearchType.java

```diff
@@ -71,6 +73,8 @@
     NESTED(JDBCType.STRUCT, null, 0, 0, false),
     OBJECT(JDBCType.STRUCT, null, 0, 0, false),
     DATE(JDBCType.TIMESTAMP, Timestamp.class, 24, 24, false),
```

Comment:
DATE to TIMESTAMP?

Reply:
DATE to DATE breaks the test by failing to load the `kibana_sample_data_flights` index. So I temporarily changed it back to TIMESTAMP. Will look into the issue